ui/src/components/chat/group_users.rs

```diff
@@ -11,12 +12,14 @@ use kit::{
     components::user_image::UserImage,
     elements::input::{Input, Options},
 };
-use warp::logging::tracing::log;
+use warp::{crypto::DID, logging::tracing::log};
 
 #[derive(Props, Eq, PartialEq)]
 pub struct Props {
     #[props(!optional)]
     active_chat: Option<Chat>,
+    #[props(!optional)]
+    group_creator: bool,
```

Comment:
Where is this used at?

Reply:
it wasn't, thanks, it was from a bigger pr i was working on that i removed stuff from for this PR and missed it